docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -1003,6 +1003,7 @@ Valid options for `[Pod]` are listed below:
 | UIDMap=0:10000:10                   | --uidmap=0:10000:10                    |
 | UserNS=keep-id:uid=200,gid=210      | --userns keep-id:uid=200,gid=210       |
 | Volume=/source:/dest                | --volume /source:/dest                 |
+| HostName=name                       | --hostname=name                        |
```

Comment:
Build is complaining that the table should be sorted alphabetically

Reply:
oh okay